checker/jtreg/nullness/defaultsPersist/Fields.java

```diff
@@ -4,6 +4,7 @@
  *
  * @compile ../PersistUtil.java Driver.java ReferenceInfoUtil.java Fields.java
  * @run main Driver Fields
+ * @ignore This is fails for Java 11.
```

Comment:
@wmdietl Can you take a look at this failure?  The bytecode looks to have the same annotations for both Java 8 and Java 11, but it fails on with Java 11

Reply:
I've opened #2816 for this. I won't get to it before the November release.